architecture/README.md

```diff
@@ -0,0 +1,654 @@
+# Search Service Architecture
+
+Source of truth for the application architecture of the ONS search service including backend processing of data. C4 diagrams available via [ONS google drive](https://drive.google.com/drive/folders/1usE1C-VzQIrymSJ7tJzXxXjZAmXLge2S).
```

Comment:
This is the old google drive Nathan

Reply:
Correct one here: https://drive.google.com/drive/folders/15Qq3wgaULer96kXGuMDUo6D0-vAz7um1